ts/javascript/mod.ts

```diff
@@ -94,7 +93,6 @@ export function mkNpmProject(args: {
       defaultEnvironment: devShell,
     },
     {
-      devShell,
```

Comment:
Maybe worth mentioning this change in the changelog?

Reply:
Done in 4a6d1470.